meow/frontend/src/components/EditPost/Sidebar.js

```diff
@@ -1,9 +1,16 @@
 import React from "react";
-import { Calendar, TimePicker, Button } from "antd";
+import { Calendar, TimePicker, Button, notification } from "antd";
 import moment from "moment";
 import "./Sidebar.css";
 
 class Sidebar extends React.Component {
+  openNotification = () => {
+    notification.open({
+      message: "Notification Title",
+      description: "This is sadsa"
+    });
+  };
```

Comment:
Its totally ok to do this for testing but we don't want testing code to get into the master branch so please remove this

Reply:
sorry I should've checked it one more time